src/components/table/table-header/table-header.style.js

```diff
@@ -0,0 +1,78 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import { isClassic } from '../../../utils/helpers/style-helper';
+import { THEMES } from '../../../style/themes';
+
+const StyledTableHeader = styled.th`
+  ${({
+    align, sortable, theme, size
+  }) => {
+    const { table, colors, name } = theme;
+    return css`
+      background-color: ${table.header};
+      border-bottom: 1px solid ${table.secondary};
+      border-left: 1px solid ${colors.border};
+      border-right: none;
+      border-top: none;
+      box-sizing: border-box;
+      color: ${colors.white};
+      font-weight: 700;
+      height: ${isClassic(theme) || name === THEMES.base ? table.sizes.medium : table.sizes[size]};
+      outline: medium none;
+      padding: 0 8px;
+      position: relative;
+      text-align: ${align !== '' ? align : 'left'};
+      text-overflow: ellipsis;
+      user-select: none;
+      vertical-align: middle;
+      white-space: nowrap;
+
+      &:first-child {
+        border-left: none;
+      }
+
+      ${sortable && applySortableStyling(align, colors, table)}
+    `;
+  }}
+`;
+
+function applySortableStyling(align, colors, table) {
+  return css`
+    cursor: pointer;
+
+    &:hover {
+      background-color: ${table.tertiary};
+    }
+
+    a {
+      &:link,
+      &:visited,
+      &:hover,
+      &:active {
+        color: ${colors.white};
+        text-decoration: none;
+      }
+
+      span {
+        float: ${align === 'right' ? 'left' : 'right'};
```

Comment:
if `align` is not defined it will be floated right. Is this a right behaviour?

Reply:
That would be the expected default behaviour, I've added `align` to the defaultProps so this isn't going to be an issue either way now